zillow_crawler/main.py

```diff
@@ -68,7 +83,8 @@ def main():
         # ]
         # live.update(Panel(Columns(renderables, equal=True), title="Books"))
         session.close()
-
+        print("\nTotal crawl time: "+str(f'{total_crawl_time:0.2f} sec'))
```

Comment:
Format the code by running black on it.  It's installed in the repo, so just run it like `black .` in the terminal.
Try to avoid combining string concatenation with f-strings.  You can accomplish what you want with f-strings alone.
You do not have to cast the f-string as a string using `str()`.  This will happen regardless.